src/lib/types.ts

```diff
@@ -1,44 +1,46 @@
 export interface IExpert {
-    status: ExpertStatus,
-    firstName: string,
-    secondName: string,
-    email: string,
-    phone: string,
+  status: ExpertStatus;
```

Comment:
is that new rule introduced for using `;` in the interfaces?

if we don't have that one, pls search for the elsint rule so that it will be consistent for all people

Reply:
We discussed and find out that this is a correct rule to use `;` in the interfaces